mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -491,14 +491,16 @@ private boolean userHasLookupRightsOn(Mailbox mailbox, MailboxSession session) t
             LOGGER.warn("Ignoring mailbox with empty name");
         } else {
             MailboxPath sanitizedMailboxPath = mailboxPath.sanitize(mailboxSession.getPathDelimiter());
-            if (mailboxExists(sanitizedMailboxPath, mailboxSession))
+            if (mailboxExists(sanitizedMailboxPath, mailboxSession)) {
                 throw new MailboxExistsException(sanitizedMailboxPath.asString());
+            }
             // Create parents first
             // If any creation fails then the mailbox will not be created
             // TODO: transaction
             List<MailboxId> mailboxIds = new ArrayList<>();
             for (MailboxPath mailbox : sanitizedMailboxPath.getHierarchyLevels(getDelimiter()))
 
+            {
```

Comment:
on previous line

Reply:
GitHub issue, done in de91ad36202ce8de0596b3cd121aec00cd8af01f